packages/compartment-mapper/test/scaffold.js

```diff
@@ -84,6 +84,7 @@ export function scaffold(
     tags = undefined,
     searchSuffixes = undefined,
     commonDependencies = undefined,
+    additionalOptions = {},
```

Comment:
The problem this tries to solve is I wanted to avoid exposing more and more options from load/import functions up to the scaffold. I'm trying to make it possible to pass any options to all calls scaffold makes without adding too many new options to an already complex scaffold API.